app/services/UserService.py

```diff
@@ -31,3 +31,21 @@ async def modify_user(user_id: int, model: PydanticUserModify):
 
     except ValueError as e:
         raise HTTPException(status_code=409, detail=str(e)) from e
+
+async def get_all_users():
+    """
+    Retrieves all users.
+    """
+    users = await UserInDB.all()
+    return [PydanticUserResponse(**user.__dict__) for user in users]
+
+
+async def get_user_by_id(user_id: int):
+    """
+    Retrieves a user by their ID.
+    """
+    user = await UserInDB.get_or_none(id=user_id)
+    if user is None:
+        raise HTTPException(status_code=404, detail=CommonErrorMessages.USER_NOT_FOUND)
+    
+    return PydanticUserResponse(**user.__dict__)
```

Comment:
Missing empty line at the end of the file.
Also, use the from_orm method, as suggested above.

Reply:
done